plugins/newrelic-dashboard/src/plugin.ts

```diff
@@ -91,3 +91,23 @@ export const DashboardSnapshotComponent = newRelicDashboardPlugin.provide(
     },
   }),
 );
+
+/**
+ * Render a dashboard snapshots list from Newrelic in backstage. Use dashboards which have the tag `isDashboardPage: true`
+ *
+ * @remarks
+ * This can be helpful for rendering dashboards outside of Entity Catalog.
+ *
+ * @public
+ */
+export const DashboardSnapshotListComponent = newRelicDashboardPlugin.provide(
```

Comment:
We don't suffix with `Component` on components.

Reply:
ok

Reply:
as you can see the DashboardSnapshot above has the suffix
should I fix it as well?

Reply:
also this:
```ts
export const EntityNewRelicDashboardCard = newRelicDashboardPlugin.provide(
  createComponentExtension({
    name: 'EntityNewRelicDashboardListComponent',
```